documentation/src/components/LeftSidebar/LeftSidebar.astro

```diff
@@ -19,142 +19,184 @@ const currentPageMatch = currentPage.endsWith("/") ? currentPage.slice(0, -1) :
 
 const { slug, lang } = getPathParamsFromId(Astro.url.pathname);
 const baseSidebar = await getLocalizedSidebar(slug, lang);
-// TODO: add collapsible groups
 ---
 
 <nav aria-labelledby="grid-left">
   <ul class="nav-groups">
-    {
-      baseSidebar.map((group) => (
-        <li>
-          <div class="nav-group">
-            <h2 class="nav-group-title">{group.title}</h2>
-            <ul>
-              {group.items.map((item) => (
-                <li class="nav-link">
-                  <a
-                    href={item.link}
-                    aria-current={currentPageMatch === item.link ? "page" : false}
-                    target={item.link.startsWith("/") ? undefined : "_blank"}
-                  >
-                    {item.title}
-                  </a>
-                </li>
-              ))}
-            </ul>
-          </div>
-        </li>
-      ))
-    }
+    {baseSidebar.map((group, index) => (
+      <li>
+        <div class="nav-group">
+          <button
+            class="nav-group-title"
+            aria-expanded="true"
+            aria-controls={`group-${index}`}
+          >
+            {group.title}
+            <svg xmlns="http://www.w3.org/2000/svg" class="arrow" width="18" height="18" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round"> <path stroke="none" d="M0 0h24v24H0z" fill="none"></path> <polyline points="9 6 15 12 9 18"></polyline> </svg>
```

Comment:
Would you like to move this icon to Chevron.astro at https://github.com/effector/effector/tree/master/documentation/src/icons ?

Reply:
resolved!